src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -63,7 +64,7 @@ public String getDisplayName() {
             return null;
         }
 
-        StringJoiner joiner = new StringJoiner(MainTableFormat.COL_DEFINITION_FIELD_SEPARATOR);
+        StringJoiner joiner = new StringJoiner(JabRefPreferences.COL_DEFINITION_FIELD_SEPARATOR);
```

Comment:
The code is wrong anyway and somehow slipped my attention, thanks for noticing! It should have been `Globals.prefs.getValue(JabRefPreferences.COL_DEFINITION_FIELD_SEPARATOR)` of course. 

Regarding the migration of constants into preferences: that is what *you* asked me to do in #1408 ;-) However, it is no hassle to move this particular constant back into `GUIGlobals`. 

Will do, as with the rest of your comments